pkg/cmd/roachtest/testselector/snowflake_query.sql

```diff
@@ -8,20 +8,21 @@ with builds as (
   where
     start_date > dateadd(DAY, ?, current_date()) -- last "forPastDays" days
     and lower(status) = 'success' -- consider only successful builds
-    and branch_name = 'master' -- consider only builds from master branch
+    and branch_name = ? -- consider the builds from target branch
     and lower(name) like ? -- name is based on the suite and cloud e.g. '%roachtest nightly - gce%'
   group by 1
 ), test_stats as (
   -- for all the build IDs select do a inner join on all the tests
   select test_name, -- distinct test names
-         count(case when duration>0 then test_name end) as total_runs, -- all runs with duration>0 (excluding all ignored test runs)
+         count(case when duration>0 and status='SUCCESS' then test_name end) as total_runs, -- all successful runs with duration>0
```

Comment:
`total_runs` is now the count of successful runs; but we also have `success_count` below. The only difference seems to be the inclusion of `duration > 0`. Do we need both?

Sorry if I already asked this, but remind me: do we actually have rows where `duration = 0` and status is `SUCCESS`? If so, when would that happen?

Reply:
This makes sense. Thanks for pointing out. Having the success cound is actually good enough now!